src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentSum = 40 * days;
+
+  switch (true) {
```

Comment:
it's not a good idea to use switch case with the condition of 'true'. It's better to use a few if

Reply:
If you use switch with `true` condition, you do not need `switch` at all)